### PR TITLE
docs!: add notes on thai font where accent chars collapse

### PR DESCRIPTION
**Pull request by hackinteach**
As discussed in #459 , I have added notes on Thai accent characters.

### CODE REVIEW

File: docs/Unicode.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ more information.
 Then, to use a Unicode font in your script, pass `True` as the fourth parameter 
 of [`add_font`](fpdf/fpdf.html#fpdf.fpdf.FPDF.add_font).
 
+**Notes on Thai Fonts**
```
[the remark this answers is not in the document]
Comment (hackinteach) [reply docs/Unicode.md]: Sure, i just pushed the more general version. Not sure if that is ok. Let me know if there's still room for improvement. I'm not quite familiar with font and unicode thing but will do my best :)

**Comment by Lucas-C** [comment]
Looks great to me, thank you @hackinteach for submitting this!

I'm going to wait for @gmischler review on the latest version before merging this 😊

**Comment by Lucas-C** [comment]
@all-contributors please add @hackinteach for documentation

**Comment by gmischler** [comment]
There's one sentence that could be phrased a bit more clearly. Other than that, a good summary of the issue and the solution!

**Comment by Lucas-C** [comment]
Merging this now.
Thank you both @gmischler & @hackinteach!

**Pull request merged**